src/components/FilterSearch.tsx

```diff
@@ -30,7 +30,7 @@ const builtInCssClasses: Readonly<FilterSearchCssClasses> = {
   inputElement: 'text-sm bg-white outline-none h-9 w-full p-2 rounded-md border border-gray-300 focus:border-primary text-neutral-dark placeholder:text-neutral',
   sectionLabel: 'text-sm text-neutral-dark font-semibold pb-2',
   focusedOption: 'bg-gray-100',
-  option: 'text-sm text-neutral-dark pb-1 cursor-pointer'
+  option: 'text-sm text-neutral-dark pb-1 cursor-pointer hover:bg-gray-100'
```

Comment:
Is there a way we can apply the styling defined in focusedOption on hover? Right now if somebody has a custom focusedOption class the hover styling will be different. This isn't crucial so don't spend forever if it seems impossible

Reply:
HOWEVER I suspect it's possible

Reply:
Actually hmm, on second thought I'm not sure if we want to enforce the same styling for both highlighting and on hover

Reply:
OK--I think that it might be a good idea, but we can address it in a later item if it comes up as something we want.

Reply:
* note that if we do end up wanting to change it, we should also change the behavior to match in the SearchBar dropdown.